docs/user_manual/llm.md

```diff
@@ -0,0 +1,132 @@
+# OpenAI Integration
+
+Assemblyline now supports LLM integration using the OpenAI chat completions API.
+
+## Current Features of the AI Integration
+
+In its current state, it will allow Assemblyline users to have the LLM:
+
+  1. Create an hybrid report where the summary of the finding and the detailled analysis is generated by the LLM and the associated IOCs and extracted files statically extracted from the API
+  2. Create an executive summary of the detail analysis of a submission or a file
+  3. Analyse code and provide a summary and detailled analysis of what that code does
+
+### Hybrid reports
+
+Now as part of the report view of Assemblyline, a button was added at the top let you use the LLM generated report.
+
+![Report button](./images/ai_report_button.png)
+
+By clicking the button, Assemblyline will bundle the Assemblyline report in compact format and ask the LLM to create an executive summary followed by a detailled analysis of the submission. At the end of the detaillled analysis, you will find a "**Powered by AI**" watermark that indicate that this section was generated by the LLM.
+
+![Hybrid Report](./images/ai_report.png)
+
+### Executive Summaries
+
+While viewing either the detail of a submission or the detail of a file, when the AI Integration is turned on, the system will automatically ask the LLM to generate executive summaries of the submission or file that you are currently viewing.
+
+These executive summary will be in their own section and will be followed by the same "**Powered by AI**" watermark found in the report so you can easily spot that these are not generated by Assemblyline.
```

Comment:
```suggestion
These executive summaries will be in their own section and will be followed by the same "**Powered by AI**" watermark found in the report so that you can easily spot that this data is not generated by Assemblyline.
```